lib/Doctrine/DBAL/Types/Type.php

```diff
@@ -114,12 +116,13 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform)
      * Converts a value from its database representation to its PHP representation
      * of this type.
      *
-     * @param mixed                                     $value    The value to convert.
-     * @param \Doctrine\DBAL\Platforms\AbstractPlatform $platform The currently used database platform.
+     * @param mixed                                     $value            The value to convert.
+     * @param \Doctrine\DBAL\Platforms\AbstractPlatform $platform         The currently used database platform.
+     * @param array|null                                $fieldDeclaration The field declaration.
      *
      * @return mixed The PHP representation of the value.
      */
-    public function convertToPHPValue($value, AbstractPlatform $platform)
+    public function convertToPHPValue($value, AbstractPlatform $platform, array $fieldDeclaration = null)
```

Comment:
Such BC break is a no-go in Doctrine DBAL 2.x. It would break all projects using a custom type class